api/controller/questionController.js

```diff
@@ -0,0 +1,98 @@
+import db from '../db/index';
+
+
+export function getAllQuestions(req, res) {
```

Comment:
Again, no `function`. Make use of arrow-function. Also, document your methods using JSDoc